firmware/base2015/PCLink.cpp

```diff
@@ -0,0 +1,57 @@
+#include <string>
+#include "PCLink.hpp"
+
+PCLink::PCLink(void)
```

Comment:
I think you can delete this constructor.  Because the other one has default values for the parameters, the result should be the same.